app/assets/stylesheets/application.scss

```diff
@@ -105,3 +137,102 @@ body {
   width: 250px;
   padding: 5px;
 }
+
+// Masonary
+* {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+
+// Grid
+
+.grid {
+  background: $mediumaquamarine;
+  max-width: 2000px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+// clearfix
+.grid:after {
+  content: "";
+  display: block;
+  clear: both;
+}
+
+// Grid-items
+.trip-item {
+  background: $darkseagreen;
+  border: 2px solid #333;
```

Comment:
Color literals like `#333` should only be used in variable declarations; they should be referred to via variable everywhere else.